src/css/controls/imports/modal.less

```diff
@@ -0,0 +1,32 @@
+.jw-modal {
+    width: 284px;
+
+    .jw-breakpoint-7 &,
+    .jw-breakpoint-6 &,
+    .jw-breakpoint-5 & {
+        .rect(284px, 232px);
+        max-height: 232px;
+    }
+
+    .jw-breakpoint-4 &,
+    .jw-breakpoint-3 & {
+        .rect(284px, 192px);
+        max-height: 192px;
+    }
+
+    .jw-breakpoint-2 & {
+        .rect(284px, 179px);
+        max-height: 179px;
+    }
+
+    .jw-breakpoint-2 &,
+    .jw-flag-small-player & {
+        bottom: 0;
+        right: 0;
+        height: 100%;
+        width: 100%;
+        max-height: none;
+        max-width: none;
+        z-index: 1;
```

Comment:
we avoid setting z-index to avoid conflicts with customer's layers
 do you really need this ?

Reply:
Unfortunately, we use z-index: 1 all over the place, so I do have to.

![image](https://user-images.githubusercontent.com/2903955/53578723-2cfc6a00-3b46-11e9-9d7e-571e56605dac.png)

Reply:
My bad, you are right! We should avoid using z-index >1, as you have probably already noticed! Great catch!